server/irmaserver/helpers.go

```diff
@@ -804,3 +806,58 @@ func (s *Server) newSession(
 
 	return ses, nil
 }
+
+func (session *sessionData) generateSdJwts(privKey *ecdsa.PrivateKey, issuerUrl string) ([]*sdjwtvc.SdJwtVc, error) {
+	// Check that the request is a valid issuance request
+	req, err := session.getRequest()
+	if err != nil {
+		return nil, err
+	}
+
+	issuanceReq, ok := req.(*irma.IssuanceRequest)
+	if !ok {
+		return nil, errors.New("session request is not an issuance request; cannot generate SD-JWTs")
+	}
+
+	if !issuanceReq.RequestSdJwts {
+		return nil, nil // No SD-JWTs requested, return nil
+	}
+
+	creator := sdjwtvc.DefaultEcdsaJwtCreator{
+		PrivateKey: privKey,
+	}
+
+	// An issuance request may contain multiple credentials, so we need to create a separate SD-JWT for each one
+	sdJwts := make([]*sdjwtvc.SdJwtVc, len(issuanceReq.Credentials))
+
+	for c, cred := range issuanceReq.Credentials {
+		disclosures := make([]sdjwtvc.DisclosureContent, len(cred.Attributes))
+		i := 0
+		for attrKey, attrVal := range cred.Attributes {
+			disclosure, err := sdjwtvc.NewDisclosureContent(attrKey, attrVal)
+
+			if err != nil {
+				return nil, err
+			}
+
+			disclosures[i] = disclosure
+			i++
+		}
+
+		b := sdjwtvc.NewSdJwtVcBuilder().
+			WithHashingAlgorithm(sdjwtvc.HashAlg_Sha256).
```

Comment:
Not really concerning this PR, but it seems strange to choose the hashing algorithm but not choosing the signature scheme.